migrations/20161013143349_Projects.js

```diff
@@ -0,0 +1,37 @@
+
+exports.up = function(knex, Promise) {
+  return Promise.all([
+    knex.schema.createTable('projects', function(table){
+      table.increments('id').primary()
+      table.string('title')
+      table.string('description')
+      table.string('url')
+      table.date('date')
+    }),
+    knex.schema.createTable('photos', function(table){
+      table.increments('id').primary()
+      table.string('url')
+      table.string('caption')
+      table.integer('projects_id').references('projects.id')
+    }),
+    knex.schema.createTable('teammember', function(table){
```

Comment:
Please rename table to `team_members`. Don't forget to make the same change in the `dropTable` calls below.